napari/components/dims.py

```diff
@@ -427,8 +448,18 @@ def _roll(self):
         """Roll order of dimensions for display."""
         order = np.array(self.order)
         nsteps = np.array(self.nsteps)
-        order[nsteps > 1] = np.roll(order[nsteps > 1], 1)
```

Comment:
I think this means that unitary dimensions are not rollable. If we define a `rollable: Tuple[bool, ...]` we probably want to absorb that logic into that, which could simplify the code here.